persistent/Database/Persist/Sql/Migration.hs

```diff
@@ -200,7 +200,7 @@ addMigration
     -> Sql
     -- ^ A 'Text' value representing the command to run on the database.
     -> Migration
-addMigration isSafe sql = lift (tell [(isSafe, sql)])
+addMigration isSafe sql = lift (tell [(not isSafe, sql)])
```

Comment:
This would be a breaking change in behavior. Let's instead change the documentation? That way, we won't break any existing code, and folks writing new code will be able to tell from the docs how to do it right.

Reply:
This is done now.